docsrc/installation.rst

```diff
@@ -1,17 +1,39 @@
 Installation
 ============
 
-Both CmdStanPy and CmdStan must be installed;
-since the CmdStanPy package contains utility ``install_cmdstan``,
-we recommend installing the CmdStanPy package first.
+CmdStanPy is a pure-Python3 package, but it relies on CmdStan for all
+of its functionality. There are several ways to install CmdStan and CmdStanPy, 
+which depend on the kind of user you are. 
 
 
-Install package CmdStanPy
+Conda users (Recommended)
 -------------------------
 
-CmdStanPy is a pure-Python3 package.
+If you use `conda <https://docs.conda.io/en/latest/>`__, 
+installation of both can be done very simply. CmdStanPy
+and CmdStan are both available via the 
+`conda-forge <https://conda-forge.org/>`__ repository.
 
-It can be installed from PyPI via URL: https://pypi.org/project/cmdstanpy/ or from the
+We recommend creating a new environment for CmdStan[Py]:
+
+.. code-block:: bash
+
+    conda create -n cmdstan -c conda-forge cmdstanpy
```

Comment:
This is good line (one thing to remember is to call `-c conda-forge` when creating a new environment -> C++ stuff goes wrong if this is not done, but tbh this probably doesn't affect cmdstanpy use unless there is something clashing between user installed compiler and conda-forge compiled cmdstan)

Reply:
`-c conda-forge` is part of the command you highlighted - are you suggesting that this be called out in the instructions more specifically?

Reply:
Not sure, but 50% of instructions get this correctly (by accident) and other half don't care.